server/search/hits.go

```diff
@@ -130,8 +133,17 @@ func hitsComparator(this, that *Hit, sortingOrder *sort.Ordering) int {
 		var thisV, thatV float64
 
 		switch thisVal.(type) {
-		case float32, float64, int, int64:
-			thisV, thatV = thisVal.(float64), thatVal.(float64)
+		case json.Number:
```

Comment:
may be add a comment here that we expect json.Number here as we are using json decoder.